packages/lib-classifier/src/store/DataVisAnnotatingStore.js

```diff
@@ -0,0 +1,37 @@
+import { types } from 'mobx-state-tree'
+
+const DataVisAnnotatingStore = types
+  .model('DataVisAnnotatingStore', {
+    active: types.optional(types.number, 0),
+  })
+
+  .actions(self => {
+    function afterAttach() {
+      createClassificationObserver()
+    }
+
+    function createClassificationObserver() {
+      const classificationDisposer = autorun(() => {
+        onAction(getRoot(self), (call) => {
+          if (call.name === 'completeClassification') self.reset()
```

Comment:
I just realized that this should probably reset between steps, not just at the end of the classification.

Reply:
Is there a reason that this listens to the whole tree, and not just the classifications store? Elsewhere, I've been adding listeners to a particular node rather than root.

Reply:
Nope. This bit needs improvement.

Reply:
MST warns about doing this... but our use case seems ok?